### ISSUE TITLE
[kernel_target_abort] spdk/scripts/setup.sh: line 154: echo: write error: Invalid argument

### ISSUE THREAD

**Issue by rafalste** [spdk/spdk#2854]
# CI Intermittent Failure
```
00:18:09.924  ************************************
00:18:09.924  START TEST kernel_target_abort
00:18:09.924  ************************************
00:18:09.924   13:31:59	-- common/autotest_common.sh@1064 -- # xtrace_enable
00:18:09.924   13:31:59	-- common/autotest_common.sh@1065 -- # kernel_target
00:18:09.924   13:31:59	-- target/abort_qd_sizes.sh@62 -- # local name=kernel_target
00:18:09.924   13:31:59	-- target/abort_qd_sizes.sh@64 -- # configure_kernel_target kernel_target
00:18:09.924   13:31:59	-- nvmf/common.sh@578 -- # kernel_name=kernel_target
00:18:09.924   13:31:59	-- nvmf/common.sh@579 -- # nvmet=/sys/kernel/config/nvmet
00:18:09.924   13:31:59	-- nvmf/common.sh@580 -- # kernel_subsystem=/sys/kernel/config/nvmet/subsystems/kernel_target
00:18:09.924   13:31:59	-- nvmf/common.sh@581 -- # kernel_namespace=/sys/kernel/config/nvmet/subsystems/kernel_target/namespaces/1
00:18:09.924   13:31:59	-- nvmf/common.sh@582 -- # kernel_port=/sys/kernel/config/nvmet/ports/1
00:18:09.924   13:31:59	-- nvmf/common.sh@584 -- # local block nvme
00:18:09.924   13:31:59	-- nvmf/common.sh@586 -- # [[ ! -e /sys/module/nvmet ]]
00:18:09.924   13:31:59	-- nvmf/common.sh@587 -- # modprobe nvmet
00:18:10.181   13:31:59	-- nvmf/common.sh@590 -- # [[ -e /sys/kernel/config/nvmet ]]
00:18:10.182   13:31:59	-- nvmf/common.sh@592 -- # /var/jenkins/workspace/nvmf-tcp-phy-autotest/spdk/scripts/setup.sh reset
00:18:10.182  Waiting for block devices as requested
00:18:12.081  0000:81:00.0 (8086 0a54): vfio-pci -> nvme
00:18:12.081  /var/jenkins/workspace/nvmf-tcp-phy-autotest/spdk/scripts/setup.sh: line 154: echo: write error: Invalid argument
00:18:12.081    13:32:01	-- nvmf/common.sh@592 -- # trap - ERR
00:18:12.081    13:32:01	-- nvmf/common.sh@592 -- # print_backtrace
00:18:12.081    13:32:01	-- common/autotest_common.sh@1093 -- # [[ ehxBET =~ e ]]
00:18:12.081    13:32:01	-- common/autotest_common.sh@1095 -- # args=('kernel_target' 'kernel_target' 'kernel_target_abort' '--iso' '--transport=tcp')
00:18:12.081    13:32:01	-- common/autotest_common.sh@1095 -- # local args
00:18:12.081    13:32:01	-- common/autotest_common.sh@1097 -- # xtrace_disable
00:18:12.081    13:32:01	-- common/autotest_common.sh@19 -- # '[' no '!=' yes ']'
00:18:12.081    13:32:01	-- common/autotest_common.sh@20 -- # PREV_BASH_OPTS=ehxBET
00:18:12.081    13:32:01	-- common/autotest_common.sh@21 -- # [[ ehxBET == *\x* ]]
00:18:12.081    13:32:01	-- common/autotest_common.sh@22 -- # XTRACE_DISABLED=yes
00:18:12.081    13:32:01	-- common/autotest_common.sh@24 -- # set +x
00:18:12.081  ========== Backtrace start: ==========
00:18:12.081  
00:18:12.081  in /var/jenkins/workspace/nvmf-tcp-phy-autotest/spdk/test/nvmf/common.sh:592 -> configure_kernel_target(["kernel_target"])
00:18:12.081       ...
00:18:12.081     587 			modprobe nvmet
00:18:12.081     588 		fi
00:18:12.081     589 	
00:18:12.081     590 		[[ -e $nvmet ]]
00:18:12.081     591 	
00:18:12.081  => 592 		"$rootdir/scripts/setup.sh" reset
00:18:12.081     593 	
00:18:12.081     594 		# Find nvme with an active ns device
00:18:12.081     595 		for block in /sys/block/nvme*; do
00:18:12.081     596 			[[ -e $block ]] || continue
00:18:12.081     597 			block_in_use "${block##*/}" || nvme="/dev/${block##*/}"
00:18:12.081       ...
00:18:12.081  in test/nvmf/target/abort_qd_sizes.sh:64 -> kernel_target([])
00:18:12.081       ...
00:18:12.081     59  	}
00:18:12.081     60  	
00:18:12.081     61  	kernel_target() {
00:18:12.081     62  		local name=kernel_target
00:18:12.081     63  	
00:18:12.081  => 64  		configure_kernel_target "$name"
00:18:12.081     65  		rabort "$TEST_TRANSPORT" IPv4 "$NVMF_INITIATOR_IP" "$NVMF_PORT" "$name"
00:18:12.081     66  		clean_kernel_target
00:18:12.081     67  	}
00:18:12.081     68  	
00:18:12.081     69  	nvmftestinit
00:18:12.081       ...
00:18:12.081  in /var/jenkins/workspace/nvmf-tcp-phy-autotest/spdk/test/common/autotest_common.sh:1065 -> run_test(["kernel_target_abort"],["kernel_target"])
00:18:12.081       ...
00:18:12.081     1060		timing_enter $test_name
00:18:12.081     1061		echo "************************************"
00:18:12.081     1062		echo "START TEST $test_name"
00:18:12.081     1063		echo "************************************"
00:18:12.081     1064		xtrace_restore
00:18:12.081     1065		time "$@"
00:18:12.081     1066		xtrace_disable
00:18:12.081     1067		echo "************************************"
00:18:12.081     1068		echo "END TEST $test_name"
00:18:12.081     1069		echo "************************************"
00:18:12.081     1070		timing_exit $test_name
00:18:12.081       ...
00:18:12.081  in test/nvmf/target/abort_qd_sizes.sh:80 -> main(["--transport=tcp"],["--iso"])
00:18:12.081       ...
00:18:12.081     75  	((${#nvmes[@]} > 0))
00:18:12.081     76  	
00:18:12.081     77  	nvme=${nvmes[0]}
00:18:12.081     78  	
00:18:12.081     79  	run_test "spdk_target_abort" spdk_target
00:18:12.081  => 80  	run_test "kernel_target_abort" kernel_target
00:18:12.081     81  	
00:18:12.081     82  	trap - SIGINT SIGTERM EXIT
00:18:12.081     83  	nvmftestfini
00:18:12.081       ...
00:18:12.081  
00:18:12.081  ========== Backtrace end ==========
```

## Link to the failed CI build

https://ci.spdk.io/public_build/autotest-per-patch_94046.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @rafalste. log: https://ci.spdk.io/public_build/autotest-per-patch_94046.html

**Comment by mikeBashStuff** [comment]
Please, note that https://review.spdk.io/gerrit/c/spdk/spdk/+/8458/ needs to be rebased to include the following fix: eab0c6649de7e518b0f66d402be3164e41aecbed. Closing for now.
